server/tools/cli/databases/mssql/change-database.cli

```diff
@@ -1,5 +1,5 @@
 /subsystem=datasources/data-source=KeycloakDS: remove()
-/subsystem=datasources/data-source=KeycloakDS: add(jndi-name=java:jboss/datasources/KeycloakDS,enabled=true,use-java-context=true,use-ccm=true, connection-url="jdbc:sqlserver://${env.DB_ADDR:mssql}:${env.DB_PORT:1433};databaseName=${env.DB_DATABASE:keycloak};sendStringParametersAsUnicode=false;integratedSecurity=false;user=${env.DB_USER:keycloak};password=${env.DB_PASSWORD:password};${env.JDBC_PARAMS:}", driver-name=sqlserver)
+/subsystem=datasources/data-source=KeycloakDS: add(jndi-name=java:jboss/datasources/KeycloakDS,enabled=true,use-java-context=true,use-ccm=true, connection-url="jdbc:sqlserver://${env.DB_ADDR:mssql}:${env.DB_PORT:1433};databaseName=${env.DB_DATABASE:keycloak};sendStringParametersAsUnicode=false;user=${env.DB_USER:keycloak};password=${env.DB_PASSWORD:password};${env.JDBC_PARAMS:}", driver-name=sqlserver)
```

Comment:
I am wondering if this is a compatible change. What happens if "integratedSecurity=false" is not explicitely specified. Is it the default? If not, this change might be breaking for existing users.

Reply:
I have never specified `integratedSecurity=false` in an MSSQL connection string. And from looking at the [source](https://github.com/microsoft/mssql-jdbc/blob/v8.2.2/src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java#L408), `false` is indeed the default.

Reply:
Possible properties with explanation and default values are also listed here: [setting-the-connection-properties](https://docs.microsoft.com/en-us/sql/connect/jdbc/setting-the-connection-properties?view=sql-server-ver15)

Just search for `integratedSecurity`.

I have just realized that this change should not only enable to set up NTLM, but Kerberos as well.

Reply:
Sounds good, thanks.